src/2-Circles/CircleDisplay.tsx

```diff
@@ -0,0 +1,490 @@
+import { DOMAIN } from '@env';
+import axios, {AxiosResponse, AxiosError} from 'axios';
+import React, { useEffect, useLayoutEffect, useRef, useState } from 'react';
+import { GestureResponderEvent, Image, Modal, ScrollView, StyleSheet, Text, TouchableOpacity, View, TextInput } from 'react-native';
+import { CircleAnnouncementListItem, CircleEventListItem, CircleListItem, CircleResponse } from '../TypesAndInterfaces/config-sync/api-type-sync/circle-types';
+import { PrayerRequestListItem } from '../TypesAndInterfaces/config-sync/api-type-sync/prayer-request-types';
+import { TabNavigationProps, CircleState, StackNavigationProps } from '../TypesAndInterfaces/custom-types';
+import theme, { COLORS, FONTS, FONT_SIZES } from '../theme';
+
+import { BottomTabBarHeightCallbackContext } from '@react-navigation/bottom-tabs';
+import { CircleStatusEnum } from '../TypesAndInterfaces/config-sync/input-config-sync/circle-field-config';
+import { useAppDispatch, useAppSelector } from '../TypesAndInterfaces/hooks';
+import { RootState, addCircle, removeCircle } from '../redux-store';
+import { AnnouncementTouchable, CircleTouchable, EventTouchable, Input_Field, PrayerRequestTouchable, Raised_Button } from '../widgets';
+
+// TODO: Make circle lookup page AS modal
+// Also think about a dynamic lookup system for associating circles, users, prayer requests, etc
+
+// Look into useLayoutEffect
+
+const CircleDisplay = ({navigation, route}:StackNavigationProps, status: number):JSX.Element => {
+    const dispatch = useAppDispatch();
+    const jwt = useAppSelector((state: RootState) => state.account.jwt);
+    const userID = useAppSelector((state: RootState) => state.account.userID);
+    const userCircles = useAppSelector((state:RootState) => state.account.userProfile.circleList);
+    
+    const routeProps = route.params;
+    const circleProps = routeProps.CircleProps as unknown as CircleListItem;
+
+    const RequestAccountHeader = {
+      headers: {
+        "jwt": jwt, 
+        "userID": userID,
+      }
+    }
+
+    const defaultState = {
+        circleData: {} as CircleResponse,
+        circleDataRequest: {} as CircleListItem
+    } as CircleState;
```

Comment:
Better to declare type in typescript instead of casting:
```
const defaultState:CircleState = {
        circleData:CircleResponse {},
        circleDataRequest:CircleListItem {}
    };
```
- Also, I beleve all the same information from `CircleListItem ` is inside `CircleResponse `